pandas/core/window.py

```diff
@@ -963,8 +964,19 @@ def count(self):
         not passed. In the future `raw` will default to False.
 
         .. versionadded:: 0.23.0
+    *args, **kwargs :
+        Arguments and keyword arguments to be passed into func.
 
-    \*args and \*\*kwargs are passed to the function""")
+    Returns
+    -------
+    Series or DataFrame
+        Return type is determined by the caller.
+
+    See Also
+    --------
+    Series.%(name)s : Series %(name)s.
+    DataFrame.%(name)s : DataFrame %(name)s.
```

Comment:
Sorry this took a while to review.

I don't understand this. In `_doc_template` we already have this, and we were appending to the docstrings from there. Why repeat that code here again and have it duplicated? I think makes more sense what we had before.

Reply:
@LJArendse I suppose this is related to your question above. So IIUC correctly using the same template for all funcs here will incorrectly call out that *args are not documented for skew.

Is that the only exception? If so seems like it would still be easiest to use doc_template in as many funcs as possible and simply replace for `skew` if that's the only one generating an error

Reply:
Hi @datapythonista
I agree 100% with what you are saying, ideally the code should just be defined in ```_doc_template``` and then just appended to the _docstring_ using the ```@Appender(_doc_template)```. However the ```@Appender(_doc_template)``` applied to pandas.core.window.Expanding.apply and pandas.core.window.Rolling.apply is not appending to the docstring correctly, the indentation is one level deeper, which throws the 'no return found' error. 
Modifying, i.e. fixing the ```_doc_template`` Expanding.apply and Rolling.apply, the ```_doc_template``` itself 'breaks' other methods' docstring (as we saw in the earlier commits). Hence the change to the actual ```_shared_docs['apply']``` (similarly to how the docstrings for the ```_shared_docs['sum']``` methods are constructed, i.e. it did not make use of an```@Appender```). 

I also modified the ```_doc_template``` with the same changes as in ```_shared_docs['apply']```, suggested by @WillAyd, to enrich the docstrings of the other methods using the ```_doc_template```.
I hope this answers your question? Thanks :)

Reply:
Hey @WillAyd 
One question: what does 'IIUC' mean?

No ```pandas.core.window.Expanding.skew``` is not the only exception. The following other methods will also be affected:
- ```pandas.core.window.Rolling.skew```
- ```pandas.core.window.Rolling.cov```
- ```pandas.core.window.Expanding.cov```
- ```pandas.core.window.EWN.cov```
- ```pandas.core.window.EWN.corr```

For these methods the addition of ```*args, **kwargs``` to ```_doc_template``` makes sense:
- ```pandas.core.window.Rolling.max```
- ```pandas.core.window.Expanding.max```
- ```pandas.core.window.EWN.mean```
- ```pandas.core.window.EWN.std```
- ```pandas.core.window.EWN.var```

Do you suggest we add ```*args, **kwargs``` to ```_doc_template``` and then modify **skew**, **cov**, and **corr**?

Reply:
Go with whichever route yields the least amount of changes / exceptions



> One question: what does 'IIUC' mean?

If I understand correctly